ironic/drivers/modules/drac/raid.py

```diff
@@ -134,6 +134,26 @@ def list_physical_disks(node):
         raise exception.DracOperationError(error=exc)
 
 
+def is_raid_controller(node, raid_controller_fqdd):
```

Comment:
This will need to be updated for this patch: https://review.opendev.org/#/c/653850/

Reply:
Done